test_runner/fixtures/neon_fixtures.py

```diff
@@ -1049,17 +1073,19 @@ def timeline_detail(self,
             include_non_incremental_physical_size_str = "1"
 
         res = self.get(
-            f"http://localhost:{self.port}/v1/tenant/{tenant_id.hex}/timeline/{timeline_id.hex}" +
-            "?include-non-incremental-logical-size={include_non_incremental_logical_size_str}" +
-            "&include-non-incremental-physical-size={include_non_incremental_physical_size_str}")
+            f"http://localhost:{self.port}/v1/tenant/{tenant_id.hex}/timeline/{timeline_id.hex}"
+            + f"?include-non-incremental-logical-size={include_non_incremental_logical_size_str}"
+            + f"&include-non-incremental-physical-size={include_non_incremental_physical_size_str}"
+        )
```

Comment:
This was a real bug found by `flake8`: we didn't use f-string for `include-non-incremental-physical-size` and `include-non-incremental-logical-size` GET parameters

Reply:
This can be streamlined with something like that:

```python
params = {}
if include_non_incremental_logical_size:
    params['include-non-incremental-logical-size'] = 'present'

if include_non_incremental_physical_size:
    params['include-non-incremental-physical-size'] = 'present'

res = self.get(
    f"http://localhost:{self.port}/v1/tenant/{tenant_id.hex}/timeline/{timeline_id.hex}",
    params=params,
)
```

Rust code checks only for parameter presence to avoid dealing with all possible truthy variants like "y, yes, 1" etc